src/App.tsx

```diff
@@ -32,9 +31,10 @@ import {
   createPolkabtcAPI,
   PolkaBTCAPI
 } from '@interlay/polkabtc';
+import { StatusCode } from '@interlay/polkabtc/build/interfaces';
 
 import Layout from 'parts/Layout';
-import ApplicationPage from 'pages/app/app.page';
+import Application from 'pages/Application';
```

Comment:
This `import` causes a compilation error

```suggestion
import Application from 'pages/application';
```

Reply:
Thank you for letting me know.
But the component name should be PascalCase according to React convention.
Could you please attach the error?

Reply:
The error I get is
```
Failed to compile.

./src/App.tsx
Module not found: Can't resolve 'pages/Application' in './polkabtc-ui/src'
```

Reply:
Perhaps you need to rename the file itself as well?

Reply:
Thanks.
Let me handle it.
Meanwhile could you please review the rest of the PR?

Reply:
Could you please check if it's fixed?

Reply:
It's fixed now :)